eng/versioning/set_versions.py

```diff
@@ -219,7 +222,7 @@ def prep_version_file_for_source_testing(build_type):
     with open(version_file, 'w', encoding='utf-8') as f:
         for line in newlines:
             f.write(line)
-    
+
```

Comment:
Just an FYI - These space removals are from me turning on the option to remove trailing space in VS Code. This is goodness.